tfhe/docs/fine_grained_api/shortint/operations.md

```diff
@@ -380,7 +380,7 @@ fn main() {
     let acc = server_key.generate_lookup_table_bivariate(|x,y| (x.count_ones()
         + y.count_ones()) as u64 % modulus );
 
-    let ct_res = server_key.smart_apply_lookup_table_bivariate(&mut ct_1, &mut ct_2, &acc);
+    let ct_res = server_key.apply_lookup_table_bivariate(&mut ct_1, &mut ct_2, &acc);
```

Comment:
nitpick : those don't need to be mutable anymore, let me know if we fix that now (we have had users in the past asking about those) or later e.g. when looking to fix the scaling issue/bug we have had for a while

Reply:
Fixed

Reply:
ok thanks !